hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -1,38 +1,38 @@
 # NVMe SSD Boot (BETA)
 
-NVMe (Non Volatile Memory express) is a standard for accessing Solid State Drives (SSD) via a PCI express bus (PCIe). You can connect these drives via the PCIe slot on a Compute Module (CM) IO board allowing a CM4 to boot from SSD.
+NVMe (non-volatile memory express) is a standard for accessing solid state drives (SSDs) via a PCIe bus. You can connect these drives via the PCIe slot on a Compute Module (CM) IO board, allowing a CM4 to boot from SSD.
```

Comment:
Perhaps this ought to say "Compute Module 4 IO Board", to match up with https://www.raspberrypi.org/products/compute-module-4-io-board/ ?

Reply:
Good catch - fixed.

I've also changed `right one` to `correct type` further on, since it is more professional/formal language.